frontend/dfx_react/README.md

```diff
@@ -0,0 +1,44 @@
+# dfx_react
+
+To get started, you might want to explore the project directory structure and the default configuration file. Working with this project in your development environment will not affect any production deployment or identity tokens.
+
+To learn more before you start working with dfx_react, see the following documentation available online:
+
+- [Quick Start](https://sdk.dfinity.org/developers-guide/quickstart.html)
+- [Developer's Guide](https://sdk.dfinity.org/developers-guide)
+- [Language Reference](https://sdk.dfinity.org/language-guide)
+
+## Important Note
+
+Before you run `dfx build`, you must update the `@dfinity/agent` property in the package.json.
+
+To get the location of your `@dfinity/agent`, run `echo $HOME/.cache/dfinity/versions/<your-dfx-version>/js-user-library`, which should print something like this:
```

Comment:
Is this internal documentation? We plan to publish @dfinity/agent publicly with the next release.

Reply:
Also, `echo $(dfx cache show)/js-user-library` will do the same.